Source/debug.cpp

```diff
@@ -242,44 +242,19 @@ std::string DebugCmdLevelUp(const std::string_view parameter)
 	return "New experience leads to new insights.";
 }
 
-std::string DebugCmdGetAllSpells(const std::string_view parameter)
-{
-	SetSpellLevelCheat(SPL_FIREBOLT, 8);
-	SetSpellLevelCheat(SPL_CBOLT, 11);
-	SetSpellLevelCheat(SPL_HBOLT, 10);
-	SetSpellLevelCheat(SPL_HEAL, 7);
-	SetSpellLevelCheat(SPL_HEALOTHER, 5);
-	SetSpellLevelCheat(SPL_LIGHTNING, 9);
-	SetSpellLevelCheat(SPL_FIREWALL, 5);
-	SetSpellLevelCheat(SPL_TELEKINESIS, 3);
-	SetSpellLevelCheat(SPL_TOWN, 3);
-	SetSpellLevelCheat(SPL_FLASH, 3);
-	SetSpellLevelCheat(SPL_RNDTELEPORT, 2);
-	SetSpellLevelCheat(SPL_MANASHIELD, 2);
-	SetSpellLevelCheat(SPL_WAVE, 4);
-	SetSpellLevelCheat(SPL_FIREBALL, 3);
-	SetSpellLevelCheat(SPL_STONE, 1);
-	SetSpellLevelCheat(SPL_CHAIN, 1);
-	SetSpellLevelCheat(SPL_GUARDIAN, 4);
-	SetSpellLevelCheat(SPL_ELEMENT, 3);
-	SetSpellLevelCheat(SPL_NOVA, 1);
-	SetSpellLevelCheat(SPL_GOLEM, 2);
-	SetSpellLevelCheat(SPL_FLARE, 1);
-	SetSpellLevelCheat(SPL_BONESPIRIT, 1);
-
-	return "Magic is no mystery for me.";
-}
-
-std::string DebugCmdMaxSpellLevel(const std::string_view parameter)
+std::string DebugCmdSetSpellsLevel(const std::string_view parameter)
 {
 	auto &myPlayer = Players[MyPlayerId];
 
+	int level = std::max(0, atoi(parameter.data()));
 	for (int i = SPL_FIREBOLT; i < MAX_SPELLS; i++) {
 		if (GetSpellBookLevel((spell_id)i) != -1) {
 			myPlayer._pMemSpells |= GetSpellBitmask(i);
-			myPlayer._pSplLvl[i] = 10;
+			myPlayer._pSplLvl[i] = level;
 		}
 	}
+	if (level == 0)
+		myPlayer._pMemSpells = 0;

```

Comment:
Would prefer a separate command for removing spells to keep things more consistent.
```suggestion
			myPlayer._pSplLvl[i] += level;
		}
	}
```

Reply:
I see no reason to artificially split it into 2 commands when setting spell level to 0 would be the logical way to remove spells

Reply:
You could give +1 to all spells to test something, then take away -1 to return to the original spells to not affect potential save games or further tests.

Reply:
meh, not really, spells are capped at 15, so lets say you gave yourself 15, there's no going back :D